scripts/checks.sh

```diff
@@ -46,7 +49,105 @@ function log_status()
 	jq -cn --argjson g "${1}" --arg f "${2}" --arg s "${3}" '[{"name":$f,"success":$g,"status":$s}]'
 }
 
+function test_upstream_dns()
+{
+	# force dnsmasq to print statistics to journal to read them back
+	systemctl kill -s USR1 dnsmasq
+	for i in $(journalctl -u dnsmasq | awk 'match($0, /[12]?[0-9]?[0-9]\.[12]?[0-9]?[0-9]\.[12]?[0-9]?[0-9]\.[12]?[0-9]?[0-9]/) {print substr($0, RSTART, RLENGTH)}'| sort -u); do
+		# shellcheck disable=SC2001
+		for j in "${external_fqdn}" "$(echo "${API_ENDPOINT}" | sed -e 's@^http.://@@')"; do
```

Comment:
This uses `http.` as the match, presumably to match `http` or `https`? However it'll only match `https` so it'd be clearer to be explicit with that and use:
```suggestion
		for j in "${external_fqdn}" "$(echo "${API_ENDPOINT}" | sed -e 's@^https://@@')"; do
```
(or make it work for either if that was the intention, but I don't think it can ever be just `http`)

Reply:
yep, good catch! updated to work with both.